package.json

```diff
@@ -19,18 +20,24 @@
     "codemod-cli": "^3.2.0"
   },
   "devDependencies": {
+    "@release-it-plugins/lerna-changelog": "^5.0.0",
     "coveralls": "^3.1.0",
     "eslint": "^8.6.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^26.6.3",
-    "prettier": "^2.5.1"
+    "lerna-changelog": "^2.2.0",
```

Comment:
Is `lerna-changelog` dependency necessary? I see `@release-it-plugins/lerna-changelog` [has it as a dependency already](https://github.com/release-it-plugins/lerna-changelog/blob/8a5d744139d1613af170f24fdad90dea6523b347/package.json#L35).

```suggestion
```